client/cmd/dexc/ui/config.go

```diff
@@ -19,12 +19,14 @@ import (
 )
 
 const (
-	maxLogRolls     = 16
-	defaultRPCAddr  = "localhost:5757"
-	defaultWebAddr  = "localhost:5758"
-	configFilename  = "dexc_mainnet.conf"
-	certsFilename   = "certs.json"
-	defaultLogLevel = "info"
+	maxLogRolls        = 16
+	defaultRPCAddr     = "localhost:5757"
+	defaultRPCCertFile = "dexc_rpc.cert"
+	defaultRPCKeyFile  = "dexc_rpc.key"
```

Comment:
If these are going in a dexc app data folder, the "dexc_" prefix is unnecessary.